### PR TITLE
add -project suffix to root directory

### LINKED ISSUE #36: Add '-project' to root level directory name
This way, we can avoid mistakes because the root level directory and the 'project' directory does not have the same name any longer.

### PR DESCRIPTION
**Pull request by DebVortex**
Fixed #36

### CODE REVIEW

**Comment by rm--** [comment]
Fixed #36

**Comment by rm--** [comment]
Looks fine for me.

File: cookiecutter.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "project_name": "My Project",
     "repo_name": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
+    "rootdir_name": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}-project",
```
Comment (keimlink) [inline cookiecutter.json]: Wouldn't `"{{ cookiecutter.repo_name }}-project"` be easier to read and understand?

**Comment by keimlink** [comment]
After merging #59 this is unfortunately not useful any longer.

**Pull request closed without merging**